### ISSUE TITLE
Column with CHECK Constraint

### ISSUE THREAD

**Issue by ezamelczyk** [typeorm/typeorm#701]
Is there a support in TypeORM for check contraints? Something like `price numeric CHECK (price > 0)`?
If not maybe it would be a good idea to add that support? It could look like this:
```
@Column()
@Constraint('price > 0')
price: number;
```

**Comment by pleerock** [comment]
No, it does not support it. I'll left this issue to community. Feel free tor contribute.

**Comment by golergka** [comment]
@AlexMesser tried using it, turns out it's not exported on global level. Adding `export * from "./decorator/Check";` in index.ts helped.

**Comment by AlexMesser** [comment]
fixed and released in `typeorm@0.2.0-alpha.36`. You can try it via `npm i typeorm@next`.

**Comment by arnold-almeida** [comment]
FYI for those of you who prefer the migration API 

```typescript
const driver = queryRunner.connection.driver
await queryRunner.createCheckConstraint('table', new TableCheck({
  columnNames: ['col'],
  expression: `${driver.escape('col')} > 0`
}))
```

From: https://github.com/typeorm/typeorm/blob/master/test/functional/query-runner/create-check-constraint.ts
